interface/patient_file/summary/demographics.php

```diff
@@ -291,7 +291,7 @@ function editScripts(url) {
                 {text: <?php echo xlj('Add'); ?>, close: false, id: 'addButton', class: 'btn-primary btn-sm', click: AddScript},
                 {text: <?php echo xlj('Clear'); ?>, close: false, id: 'clearButton', style: 'display:none;', class: 'btn-primary btn-sm', click: AddScript},
                 {text: <?php echo xlj('Back'); ?>, close: false, id: 'backButton', style: 'display:none;', class: 'btn-primary btn-sm', click: ListScripts},
-                {text: <?php echo xlj('Done'); ?>, close: true, id: 'doneButton', class: 'btn-secondary btn-sm'}
+                {text: <?php echo xlj('Cancel'); ?>, close: true, id: 'doneButton', class: 'btn-secondary btn-sm'}
```

Comment:
This button needs to be named cancel. I have gotten so many complaints about the word done being miss-leading. People click this button instead of the save at the top because it is where their eyes are and they are done with the prescription. They are mifted when the window closes and the prescription is not saved.